DESCRIPTION

```diff
@@ -73,7 +73,8 @@ Suggests:
     biomformat,
     reldist,
     ade4,
-    microbiomeDataSets
+    microbiomeDataSets,
+    rmarkdown
```

Comment:
Why? Thats covered via `BiocStyle` isn't it?

Reply:
There was an error that said that it should be added there

Reply:
![image](https://user-images.githubusercontent.com/60338854/114998033-34f40100-9ea9-11eb-8d2a-3a5b3dce1e45.png)

Reply:
Good catch, but "Oh damn". I hope this doesn't crash a large number of the CRAN/Bioconductor builds. I haven't received any mail regarding this from the maintainer of knitr. So lets hope I don't have any other packages to modify.